### PR TITLE
Bump bsm/rucksack

### PR DESCRIPTION
**Pull request by mxmCherry**

### CODE REVIEW

File: instrument.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ func (i *Instrumenter) StreamServerInterceptor(srv interface{}, stream grpc.Serv
 func (i *Instrumenter) instrument(name string, err error, elapsed time.Duration) {
 	errtags := extractErrorTags(err)
 	status := HTTPStatusFromError(err)
-	logger := log.WithField("status", status)
+	logger := log.L().With(zap.Int("status", status)).Sugar()
```
Comment (dim) [inline instrument.go]: no need for sugar if can we also change the message format below too:

```
logger.Info("completed", zap.String("rpc", name), zap.Int("status", status), zap.Duration("elapsed", elapsed.Seconds())
```

and:
```
loggerWithTags(logger, errtags).Error("failed", zap.String("rpc", name), zap.Error(err))
```

File: instrument.go [hunk 2]
```diff
@@ -93,11 +93,11 @@ func extractErrorTags(err error) []string {
 	return tags
 }
 
-func loggerWithTags(ent *logrus.Entry, tags []string) *logrus.Entry {
+func loggerWithTags(l *zap.SugaredLogger, tags []string) *zap.SugaredLogger {
```
Comment (dim) [inline instrument.go]: as said, we can probably get away without sugar

**Review by dim — CHANGES_REQUESTED** [review]

File: instrument.go [hunk 3]
```diff
@@ -72,7 +72,7 @@ func (i *Instrumenter) instrument(name string, err error, elapsed time.Duration)
 
 func (i *Instrumenter) recover(name string) {
 	if r := recover(); r != nil {
-		log.WithField("rpc", name).Errorf("panic: %v\n%v", r, debug.Stack())
+		log.L().With(zap.String("rpc", name)).Sugar().Errorf("panic: %v\n%v", r, debug.Stack())
```
Comment (mxmCherry) [inline instrument.go:75]: @dim probably, `log.Error("panic", zap.String("rpc", name), zap.Any("panic", r), zap.String("stack", debug.Stack()))`?

Comment (dim) [reply instrument.go:75]: yeah, that one's a bit different, I wouldn't bother too much with that one

**Review by dim — APPROVED** [review]

**Pull request merged**